### DISCUSSION TITLE
About the design colors

### DISCUSSION THREAD

**Discussion by alexander-lopez-s** [cristobal-lopez/my-portafolio#12] [General]
Is it ideal that the colors to be used in the website are the sames colors in the design? 

This is my first design draft but I might have to update the colors later. @samirm00

![Screenshot 2023-05-16 at 06 49 34](https://github.com/cristobal-lopez/my-portafolio/assets/97364283/2683101c-69b2-48f7-855e-33360a1250df)

**Comment by sammou00** [comment]
@cristobal-lopez,  Looks good to me.

**Reply by alexander-lopez-s** [reply]
@samirm00 ok thanks
